From 155f456932bfdabe444f81349f56279b5f0bdc7c Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 8 Sep 2014 13:07:31 -0400
Subject: [tools] perf/sched: Cleanup, remove unused variables in map_switch_event()

Message-id: <1410181752-25631-207-git-send-email-jolsa@redhat.com>
Patchwork-id: 91937
O-Subject: [PATCH RHEL7.1 BZ1134356 206/307] perf sched: Cleanup, remove unused variables in map_switch_event()
Bugzilla: 1134356
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1134356
https://bugzilla.redhat.com/show_bug.cgi?id=1134356

upstream
========
commit 9d372ca59bcb9339b4a34a9bf978a1fc15b68b03
Author: Dongsheng Yang <yangds.fnst@cn.fujitsu.com>
Date: Fri May 16 14:37:05 2014 +0900

description
===========
In map_switch_event(), we don't care the previous process currently,
this patch remove the infomation we get but not used.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/builtin-sched.c b/tools/perf/builtin-sched.c
index 0b4fe53..d717683 100644
--- a/tools/perf/builtin-sched.c
+++ b/tools/perf/builtin-sched.c
@@ -1266,9 +1266,8 @@ static int process_sched_wakeup_event(struct perf_tool *tool,
 static int map_switch_event(struct perf_sched *sched, struct perf_evsel *evsel,
        struct perf_sample *sample, struct machine *machine)
 {
- const u32 prev_pid = perf_evsel__intval(evsel, sample, "prev_pid"),
-    next_pid = perf_evsel__intval(evsel, sample, "next_pid");
- struct thread *sched_out __maybe_unused, *sched_in;
+ const u32 next_pid = perf_evsel__intval(evsel, sample, "next_pid");
+ struct thread *sched_in;
  int new_shortname;
  u64 timestamp0, timestamp = sample->time;
  s64 delta;
@@ -1291,7 +1290,6 @@ static int map_switch_event(struct perf_sched *sched, struct perf_evsel *evsel,
   return -1;
  }
 
- sched_out = machine__findnew_thread(machine, 0, prev_pid);
  sched_in = machine__findnew_thread(machine, 0, next_pid);
 
  sched->curr_thread[this_cpu] = sched_in;
-- 
1.7.1